apps/openmw/mwmechanics/aicombat.cpp

```diff
@@ -221,12 +221,12 @@ namespace MWMechanics
 
         if(movement.mRotation[2] != 0)
         {
-            if(zTurn(actor, osg::DegreesToRadians(movement.mRotation[2]))) movement.mRotation[2] = 0;
+            if(zTurn(actor, movement.mRotation[2])) movement.mRotation[2] = 0;
```

Comment:
Kinda inconsistent. You first assign a value in radians to movement.mRotation, which is then overwritten by zTurn with a value in degrees, or set to 0 if zTurn wasn't called. The CharacterController then goes on to use the target rotation value in degrees. 
The clean thing to do would be to use either degrees or radians within the same data structure all along.